index.html

```diff
@@ -4258,6 +4259,23 @@ <h2 id="MediaElementAudioSourceNode">
 </pre>
         <section>
           <h2>
+            MediaElementAudioSourceOptions
+          </h2>
+          <p>
+            This specifies the options to use in constructing a
+            <a><code>MediaElementAudioSourceNode</code></a>. This is required.
+          </p>
+          <dl title="dictionary MediaElementAudioSourceOptions" class="idl">
+            <dt>
```

Comment:
Should this derive from `AudioNodeOptions`? Do we know in advance that no future options added to `AudioNode` can affect this node?